### PR TITLE
Bug 1249256 - [layout](uk) Remove key by moving it to alternative character

### PR DESCRIPTION
**Pull request by artpoli**
Also removing unnecessary string in keyboard_test.js

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=154fea93b0e845214a138df721b0c87aa5754977

File: apps/keyboard/js/layouts/uk.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ Keyboards.uk = {
     alt: {
       а: '@',
       г: 'ґ',
+      і: 'ї',
       '.': ',?!;:…’'
     },
     width: 12,
```
Comment (timdream) [inline apps/keyboard/js/layouts/uk.js]: All of the rows are of 11 keys... I am not entirely sure any effect if you keep the `width` 12. If this is fine we could keep it as-is.

**Comment by stenox** [comment]
@timdream Yes, you're right. I should reduce value to 11. Will change it and reply to the bug.
Thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8633318d298bedcf0a6d628277fee2c29b6dc450

**Pull request merged**